migrations/v6.js

```diff
@@ -0,0 +1,83 @@
+    course = content.find(({ _type }) => _type === 'course');
+    if (course?._globals?._components?._graphic) {
+      courseGraphicGlobals = course._globals._components._graphic;
+      return true;
+    }
+    courseGraphicGlobals = course._globals._components._graphic = {};
+    return true;
+  });
+  mutateContent('Graphic - add scrollAriaLabel attribute', async content => {
+    courseGraphicGlobals.scrollAriaLabel = 'Use the scrollbar to pan the image left and right. {{#if _graphic.alt}}{{_graphic.alt}}{{/if}}';
+    return true;
+  });
+  mutateContent('Graphic - add longdescription attribute', async content => {
+    graphics.forEach(graphic => {
+      graphic._graphic.longdescription = '';
+    });
+    return true;
+  });
+  mutateContent('Graphic - add _isScrollable attribute', async content => {
+    graphics.forEach(graphic => {
+      graphic._isScrollable = false;
+    });
+    return true;
+  });
+  mutateContent('Graphic - add _defaultScrollPercent attribute', async content => {
+    graphics.forEach(graphic => {
+      graphic._defaultScrollPercent = 0;
+    });
+    return true;
+  });
+  mutateContent('Graphic - add scrollAriaLabel attribute', async content => {
+    const course = content.find(({ _type }) => _type === 'course');
+    if (course?._globals?._components?._graphic) return true;
+    return true;
+  });
+  checkContent('Graphic - check globals _graphic attribute', async content => {
+    if (courseGraphicGlobals === undefined) {
+      throw new Error('Graphic - globals _graphic invalid');
+    }
+    return true;
+  });
+  checkContent('Graphic - check globals scrollAriaLabel attribute', async content => {
+    if (courseGraphicGlobals?.scrollAriaLabel === undefined) {
+      throw new Error('Graphic - globals scrollAriaLabel invalid');
+    }
+    return true;
+  });
+  checkContent('Graphic - check longdescription attribute', async content => {
+    let isValid = true;
+    graphics.forEach(graphic => {
+      if (graphic._graphic?.longdescription === undefined) isValid = false;
+    });
+    if (!isValid) throw new Error('Graphic - longdescription attribute invalid');
+    return true;
+  });
+  checkContent('Graphic - check _isScrollable attribute', async content => {
+    let isValid = true;
+    graphics.forEach(graphic => {
+      if (graphic._isScrollable === undefined) isValid = false;
+    });
+    if (!isValid) throw new Error('Graphic - _isScrollable attribute invalid');
+    return true;
+  });
+  checkContent('Graphic - check _defaultScrollPercent attribute', async content => {
+    let isValid = true;
+    graphics.forEach(graphic => {
+      if (graphic._defaultScrollPercent === undefined) isValid = false;
+    });
+    if (!isValid) throw new Error('Graphic - _defaultScrollPercent attribute invalid');
+    return true;
+  });
+  updatePlugin('Graphic - update to v6.2.0', { name: 'adapt-contrib-graphic', version: '6.2.0', framework: '>=5.5' });
```

Comment:
As with previous, try to use full version numbers, and the FW version for graphic v6.2.0 is FW v5.19.1 (https://github.com/adaptlearning/adapt-contrib-graphic/blob/v6.2.0/package.json)
```suggestion
  updatePlugin('Graphic - update to v6.2.0', { name: 'adapt-contrib-graphic', version: '6.2.0', framework: '>=5.19.1' });
```